indexer/src/configs.rs

```diff
@@ -113,6 +122,9 @@ async fn get_start_block_height(opts: &Opts) -> u64 {
             last_indexed_block
         }
         StartOptions::FromLatest => final_block_height(opts).await,
+        // Since NEAR Lake stores blocks in ascending order, using 0 here forces
+        // near-lake-framework to start from the first block, i.e. genesis
+        StartOptions::FromGenesis => 0,
```

Comment:
I'm still not sure whether we should rely on this implicit behaviour, but this approach was much cleaner than getting the genesis block height here or overriding it in `main.rs`.

Reply:
IMO totally fine.